phpstorm_first_run.txt

```diff
@@ -0,0 +1,46 @@
+
```

Comment:
I don't like having this file committed in this repo because we can't share it with other jetbrains related flatpaks. I think we should still fetch https://github.com/Lctrs/jetbrains-flatpak-wrapper and reference the file in it.

Reply:
no, we should cusotmize this file for each IDE, for example in this case we should explain to install php83 extension, etc etc

Reply:
Still, you may want to install other SDKs. The explanation in the old flathub-ide-wrapper are generic enough and does mention the PHP extension.

Reply:
i think we should be more specific as possible, for example "golang" extension should not be mentioned, instead we should mention all the php extensions supported. or some other specific things about php